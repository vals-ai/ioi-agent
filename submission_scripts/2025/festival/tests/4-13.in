version https://git-lfs.github.com/spec/v1
oid sha256:a6a7e3212bcb042decdcd55600ccd2b12082a13e5a596928f899c35c979a5242
size 713
