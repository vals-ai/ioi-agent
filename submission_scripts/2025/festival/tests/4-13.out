version https://git-lfs.github.com/spec/v1
oid sha256:bf1d00917ddf987bf824d2ce2f2eb672abdc1b54bb0c7f00a49e7f75aecb03d8
size 151
